card/mod/01_core/set/all/references.rb

```diff
@@ -65,9 +65,8 @@ def update_references_out
 # insert entries in reference table
 def create_references_out
   ref_hash = {}
-  content_obj = Card::Content.new raw_content, self
-  content_obj.find_chunks(Card::Content::Chunk::Reference).each do |chunk|
-    interpret_reference ref_hash, chunk.referee_name, chunk.reference_code
+  each_reference_out do |referee_name, ref_type|
+    interpret_reference ref_hash, referee_name, ref_type
   end
```

Comment:
extracted this so that it's easier to override what a reference is in a certain set